resources/artifact-ignores.properties

```
@@ -197,6 +197,7 @@ pretest-commit = https://wiki.jenkins-ci.org/display/JENKINS/Pretest+Commit+Plug
 PUCM = https://wiki.jenkins-ci.org/display/JENKINS/pucm+plugin
 rally-bookkeeper                 # removal requested by Mike Rogers: will be Rally plugin 2.0
 rally-update-plugin-1            # renamed to rally-plugin(!)
+redhat-codeready-dependency-analysis = https://plugins.jenkins.io/redhat-dependency-analytics/ # Renamed to redhat-dependency-analytics
```

```suggestion
redhat-codeready-dependency-analysis = https://github.com/jenkins-infra/repository-permissions-updater/issues/3503#issue-1870513141
```

The plugin docs do not explain the reason for deprecation; the hosting request (sort of) does by mentioning

> Red Hat Dependency Analytics plugin will substitute the existing [redhat-codeready-dependency-analysis-plugin](https://github.com/jenkinsci/redhat-codeready-dependency-analysis-plugin) since the old backend is being retired and the ownership of the project changed hands.

Alternatively,
```suggestion
redhat-codeready-dependency-analysis = https://github.com/jenkins-infra/update-center2/pull/740
```

---

Let's use the latter then